source/shared/cpp/ObjectModel/ActionParserRegistration.cpp

```diff
@@ -32,19 +32,15 @@ AdaptiveSharedNamespaceStart
         }
     }
 
-    void ActionParserRegistration::RemoveParser(std::string elementType)
+    void ActionParserRegistration::RemoveParser(std::string const &elementType)
     {
-        if (m_knownElements.find(elementType) == m_knownElements.end())
```

Comment:
The intent here is to not allow callers to remove parsers for the known element types. We want to remove the parser only if they type is not in m_knownElements.

Reply:
Thanks for checking this and for helping me understand. I will revert that part.

Reply:
Got the original code back on https://github.com/Microsoft/AdaptiveCards/pull/1448/commits/d3017a3c0398ee43c18c772a960202d5738e40b3 ; please let me know if that's alright.